### PR TITLE
[libpng] fix unused CMake variable warning

### PR DESCRIPTION
**Pull request by Pospelove**

### CODE REVIEW

**Comment by cenit** [comment]
libpng license identifier is… Libpng !!
https://spdx.org/licenses/Libpng.html

**Comment by Pospelove** [comment]
not sure libpng is a supported license name in vcpkg

**Comment by mathisloge** [comment]
vcpkg supports spdx identifiers

File: ports/libpng/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "libpng",
   "version": "1.6.37",
-  "port-version": 16,
+  "port-version": 17,
   "description": "libpng is a library implementing an interface for reading and writing PNG (Portable Network Graphics) format files",
   "homepage": "https://github.com/glennrp/libpng",
```
Comment (Cheney-W) [inline ports/libpng/vcpkg.json:6]:
```suggestion
  "homepage": "https://github.com/glennrp/libpng",
  "license": "libpng-2.0",
```

**Review by Cheney-W — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
@Osyotr why? now that we don't build everything (thanks @dg0yt) I don't see the point in waiting.

**Comment by Neumann-A** [comment]
Because a lot of port depend on libpng for a more a less cosmetic change.

**Comment by strega-nil-ms** [comment]
fair enough; marking info:next-rollup for now.

**Review by strega-nil-ms — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Included by #24211; thanks @Pospelove !

**Comment by Pospelove** [comment]
Nice !!

**Pull request closed without merging**